docs/deployment.md

```diff
@@ -4,7 +4,7 @@ All members of the Find development team are able to deploy into any of the envi
 
 ## 1. Check what you're deploying
 
-Go to the Find Build pipelines [Frontend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29) & [Backend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=46)  and identify the commit you want to deploy. It is advisable to make note of the build id at this stage. 
+Go to the Find Build pipelines [Frontend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29) & [Backend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=46)  and identify the commit you want to deploy. It is advisable to make note of the build number (Build #) at this stage. 
```

Comment:
Sorry, but I still wouldn't immediately recognise that. Can we make it clearer by calling it the "git commit SHA"?

```suggestion
Go to the Find Build pipelines [Frontend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29) & [Backend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=46)  and identify the commit you want to deploy. It is advisable to make note of the build number (which is the git commit SHA) at this stage.
```

Reply:
There is a column called Build # on each build pipeline page (i.e. https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29)